rust/benches/benches/models/serialize_models.rs

```diff
@@ -0,0 +1,48 @@
+#![allow(non_snake_case)]
+
+mod utils;
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use std::time::Duration;
+
+use xaynet_core::mask::{IntoPrimitives, Model};
+
+fn serialize(model: &Model) -> Vec<i32> {
+    model
+        .clone()
```

Comment:
We don't want to clone were. Instead we can call `to_primitives()` which doesn't consume the model.

Reply:
ok thanks, will do

Reply:
changed.